File: python/Lab_5/Lab_5/TParallelepiped.py
import Tbody as f
import random as r
class TParallelepiped(f.TBody):
    def __init__(self):
        self.a=1 + (r.random() * (50 - 1))
        self.b=1 + (r.random() * (50 - 1))
        self.c=1 + (r.random() * (50 - 1))
    def getA(self):
        return self.a
    def getB(self):
        return self.b
    def getC(self):
        return self.c
    def setA(self,a):
        self.a=a
    def setB(self,b):
        self.b=b
    def setC(self,c):
        self.c=c
    def square_of_surface(self):
        s=self.a*self.b*2+self.a*self.c*2+self.b*self.c*2
        return s
    def volume(self):
        v=self.a*self.b*self.c
        return v

File: c++/Lab5(31)/Lab5(31)/TParallelepiped.cpp
#include <iostream>
#define _USE_MATH_DEFINES 
#include <math.h>
#include "TParallelepiped.h";

TParrallelepiped::TParrallelepiped() {
	this->a = 1 + (double)rand() / RAND_MAX * 50;
	this->b = 1 + (double)rand() / RAND_MAX * 50;
	this->c = 1 + (double)rand() / RAND_MAX * 50;
}
double TParrallelepiped::getA() {
	return this->a;
}
void TParrallelepiped::SetA() {
	this->a = a;
}
double TParrallelepiped::getB() {
	return this->a;
}
void TParrallelepiped::SetB() {
	this->a = a;
}
double TParrallelepiped::getC() {
	return this->a;
}
void TParrallelepiped::SetC() {
	this->a = a;
}
double TParrallelepiped::square_of_surface() {
	double s = this->a * this->b * 2 + this->a * this->c * 2 + this->b * this->c * 2;
	return s;
}
double TParrallelepiped::volume() {
	double v = this->a * this->b * this->c;
	return v;
}